harness/determined/common/experimental/determined.py

```diff
@@ -54,33 +54,24 @@ def __init__(
             explicit_noverify=noverify,
         )
 
-        auth = authentication.Authentication(self._master, user, password, cert=cert)
+        utp = authentication.login_with_cache(self._master, user, password, cert=cert)
         retry = api.default_retry()
-        self._session = api.Session(self._master, user, auth, cert, retry)
-        token_user = auth.token_store.get_active_user()
-        if token_user is not None:
-            self._token = auth.token_store.get_token(token_user)
-        else:
-            self._token = None
+        self._session = api.Session(self._master, utp, cert, retry)
```

Comment:
I can put it in a comment if you'd like?  But not a docstring.  The docstring you are describing sounds developer-focused, not user-focused.  But docstrings on this class will be exposed to users.

Reply:
Which docstrings are exposed by autodoc can be configured. In this case, I think it means adding `__init__` to `:exclude-members:` in the right place inside https://github.com/determined-ai/determined/blob/main/docs/reference/python-sdk.rst

Reply:
talked out-of-band; we don't agree fully but we will be leaving a docstring-like comment here as part of this PR.

Reply:
Actually, I investigated this.

The current docstring _does_ belong where it is, and the generated docs are correct.  If you were to put the current docstring under `__init__`, it will still be second-choice docstring after the docstring on the class.

Still I'll add a dev-focused comment to the class.

Reply:
> The current docstring does belong where it is, and the generated docs are correct.

Not speaking to the generated docs, but [our style guide](https://google.github.io/styleguide/pyguide.html#384-classes) gives the following example:

```python
class SampleClass:
    """Summary of class here.

    Longer class information...
    Longer class information...

    Attributes:
        likes_spam: A boolean indicating if we like SPAM or not.
        eggs: An integer count of the eggs we have laid.
    """

    def __init__(self, likes_spam: bool = False):
        """Initializes the instance based on spam preference.

        Args:
          likes_spam: Defines if instance exhibits this preference.
        """
        self.likes_spam = likes_spam
        self.eggs = 0
```

Reply:
I think this is a case where, for practical reasons, the behavior of sphinx trumps what the google style guide says.

Reply:
I don't know what to do about this going forward. But this PR doesn't need to be held up on that.